indico/htdocs/sass/modules/abstracts/_email_templates.scss

```diff
@@ -55,16 +55,59 @@
     }
 }
 
-.email-template-list {
-    list-style-type: none;
-    padding-left: 0;
-    width: 800px;
+#email-template-manager {
+    max-width: 800px;
+
+    .email-template-container {
+        padding: 0 0 1em 1em;
+
+        .email-template-notifications {
+            width: 800px;
+            max-height: 800px;
+            overflow: auto;
+
+            .email-template-list {
+                list-style-type: none;
+                padding-left: 0;
+                padding-right: 20px;
+
+                .email-template {
+                    margin-bottom: 10px;
```

Comment:
We don't need to add margin bottom to each `i-box`. Simply add the classes `.i-box-group.vert` to the parent element and it will take care of the spacing between them.